src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String type = "trapezoid";
```

Comment:
The `type` variable should not be static as it is a property that pertains to an instance of the class, not the class itself. Consider removing the static modifier.